scripts/sct_compute_mtr.py

```diff
@@ -72,19 +72,14 @@ def main(args=None):
     nii_mtr.save(fname_mtr)
     # sct.run(fsloutput+'fslmaths -dt double mt0.nii -sub mt1.nii -mul 100 -div mt0.nii -thr 0 -uthr 100 fname_mtr', verbose)
 
-    # if changing output file name or location, create folder with mt0.nii.gz and mt1.nii.gz files at precised location
-    if os.path.dirname(fname_mtr) != '.':
-        sct.copy(os.path.join('.', fname_mt0), os.path.dirname(fname_mtr))
-        sct.copy(os.path.join('.', fname_mt1), os.path.dirname(fname_mtr))
-
     # go to output file directory
     os.chdir(os.path.dirname(fname_mtr))
 
     # Generate output files
     sct.printv('\nGenerate output files...', verbose)
-    sct.generate_output_file(os.path.join('.', os.path.basename(fname_mtr)), '.')
+    sct.generate_output_file(os.path.join('.', fname_mtr), '.')
```

Comment:
why not simply removing `generate_output_file`? (it has no use because the file is already where it should be)

Reply:
That would make sense. Changes applied in commit [e6ecad8](https://github.com/YangHee-Min/spinalcordtoolbox/commit/e6ecad88dc7961055d37b44479d1e598aea5b4ef)